setup.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+import setuptools
+
+# This will fail if something happens or if not in a git repository.
+# This is intentional.
+ret = subprocess.run("git describe --tags --abbrev=0", stdout=subprocess.PIPE,
+                     stderr=subprocess.PIPE, check=True, shell=True)
+version = ret.stdout.decode("utf-8").strip()
+
+with open("readme.md", 'r') as readme:
```

Comment:
```suggestion
with open("readme.md", 'r', encoding="utf-8") as readme:
```

Reply:
Is there any specific reason or is it just to be safe?

Reply:
Just to be safe, windows doesn't default to utf-8. We ran into an issue in robotpy because of it.